studio/.env.example

```
@@ -0,0 +1,17 @@
+# Sanity Studio environment variables.
+# Studio only exposes vars prefixed `SANITY_STUDIO_*` to the client bundle.
+
+# Project (typically set automatically by `sanity init`)
+SANITY_STUDIO_PROJECT_ID=
+
+# Preview origins for the Presentation tool (per workspace)
+SANITY_STUDIO_PREVIEW_ORIGIN=
+SANITY_STUDIO_RWC_US_PREVIEW_ORIGIN=
+SANITY_STUDIO_RWC_INTL_PREVIEW_ORIGIN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SANITY_STUDIO_RWC_INTL_PREVIEW_ORIGIN key should go before the SANITY_STUDIO_RWC_US_PREVIEW_ORIGIN key

(UnorderedKey)